lib/sidekiq/cli.rb

```diff
@@ -269,6 +269,19 @@ def validate!
         logger.info @parser
         die(1)
       end
+
+      errors = []
```

Comment:
In favor of collecting errors, wouldn't a setting as :fail_on_first (default - true) solve this problem and make both sides happy?